src/App.vue

```diff
@@ -18,6 +18,7 @@ export default
     {
         // todo:
         // to be refactored => shall move to action
```

Comment:
There should be an action dispatched at creation to gather user data from geolocation API